hathor/transaction/storage/rocksdb_storage.py

```diff
@@ -153,6 +153,21 @@ def get_count_tx_blocks(self) -> int:
         keys_count = int(keys_bcount)
         return keys_count
 
+    def get_sst_files_sizes_by_cf(self, cfs: Optional[List[Any]] = None) -> Dict[bytes, float]:
```

Comment:
I think we could use `cfs: Optional[List['rocksdb.ColumnFamilyHandle']]` here, that's what I used in other places. In practice it will be the same as using `Any` because `rocksdb` isn't typed, but I think it's better than using `Any` directly.

Reply:
Fixed in https://github.com/HathorNetwork/hathor-core/pull/485/commits/bddd8196b7cdf1124cc30af11d8745822c19d47c